webui/react/package.json

```diff
@@ -14,6 +14,14 @@
     "lint": "npm run lint:js && npm run lint:css",
     "lint:js": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
     "lint:css": "stylelint --config .stylelintrc.js \"src/**/*.{css,less,scss}\"",
+    "prettier": "npm run prettier:js && npm run prettier:css && npm run prettier:misc",
+    "prettier:js": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
+    "prettier:css": "prettier --write \"src/**/*.{css,less,scss}\"",
+    "prettier:misc": "prettier --write \"src/**/*.{md,json}\"",
+    "prettier:check": "npm run prettier:check:js && npm run prettier:check:css && npm run prettier:check:misc",
+    "prettier:check:js": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\"",
+    "prettier:check:css": "prettier --check \"src/**/*.{css,less,scss}\"",
+    "prettier:check:misc": "prettier --check \"src/**/*.{md,json}\"",
```

Comment:
I split up prettier command, not sure if its useful tho

Reply:
The one suggestion I have here is to use the generic `format` instead of "prettier"

Reply:
Approved after this change